lib/async_hooks.js

```diff
@@ -477,7 +477,7 @@ function runCallback(cb, asyncId) {
 module.exports = {
   // Public API
   createHook,
-  currentId,
+  currentAsyncId,
```

Comment:
I know we’re okay with some breakage as long as this is fresh & experimental, but maybe
`Object.defineProperty(module.exports, 'currentId', { value: currentAsyncId });`?